File: Lista_2/Exercicios_SALA-lista-2/EX8-Divisao_2_3-s/EX8-s.py
n = []
n.append(int(input()))
n.append(int(input()))
n.append(int(input()))
L2 = []
L3 = []
for i in n:
    if i % 2 == 0:
        L2.append(i)
    if i % 3 == 0:
        L3.append(i)
print(f"números divisíveis por 2: {L2}")
print(f"números divisíveis por 3: {L3}")

File: Lista_2/Exercicios_SALA-lista-2/EX8-Divisao_2_3-s/EX8-s.c
#include <stdio.h>

int main() {
    int n1, n2, n3;
    scanf("%d", &n1);
    scanf("%d", &n2);
    scanf("%d", &n3);

    if (n1 % 2 == 0) {
        printf("%d é divisível por 2\n", n1);
    }
    if (n1 % 3 == 0) {
        printf("%d é divisível por 3\n", n1);
    }
    if (n2 % 2 == 0) {
        printf("%d é divisível por 2\n", n2);
    }
    if (n2 % 3 == 0) {
        printf("%d é divisível por 3\n", n2);
    }
    if (n3 % 2 == 0) {
        printf("%d é divisível por 2\n", n3);
    }
    if (n3 % 3 == 0) {
        printf("%d é divisível por 3\n", n3);
    }

    return 0;
}
